roles/hci_prepare/tasks/common.yml

```
@@ -0,0 +1,22 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Set common facts
+  ansible.builtin.set_fact:
+    __cifmw_hci_prepare_namespace: >-
```

```suggestion
    _cifmw_hci_prepare_namespace: >-
```

You have your brain in python mode :P

---

Fixed.